client/layout/masterbar/style.scss

```diff
@@ -84,11 +84,23 @@ $autobar-height: 20px;
 }
 
 .masterbar__item-logo {
-	font-size: 17px;
-	font-weight: 300;
+	padding-left: 0;
 
-	.tld {
-		color: rgba(255, 255, 255, 0.6);
+	svg {
```

Comment:
Can we add a classname for this one just to avoid selecting all svgs in the page (which with gridicons all over the place will be quite a bit). Something like `.masterbar__wpcom-logo`.